dags/inference/configs/trt_llm_mlperf_v40_config.py

```diff
@@ -0,0 +1,153 @@
+
+RUNTIME_IMAGE = RuntimeVersion.TPU_UBUNTU2204_BASE.value
+GCS_SUBFOLDER_PREFIX = test_owner.Team.INFERENCE.value
+
+
+def get_trt_llm_mlperf_v40_gpu_config(
+    machine_type: vm_resource.MachineVersion,
+    image_project: vm_resource.ImageProject,
+    image_family: vm_resource.ImageFamily,
+    accelerator_type: vm_resource.GpuVersion,
+    count: int,
+    gpu_zone: vm_resource.Zone,
+    time_out_in_min: int,
+    test_name: str,
+    project: Project,
+    network: str,
+    subnetwork: str,
+    model_configs: Dict = {},
+) -> task.GpuCreateResourceTask:
+  docker_container_name = "mlperf-inference"
+  set_up_cmds = (
+      # Install Nvidia driver
+      "wget -c https://us.download.nvidia.com/tesla/550.54.15/NVIDIA-Linux-x86_64-550.54.15.run",
+      "chmod u+x NVIDIA-Linux-x86_64-550.54.15.run",
+      "sudo ./NVIDIA-Linux-x86_64-550.54.15.run -x-module-path=/usr/lib/xorg/modules --ui=none -x-library-path=/usr/lib -q",
+      "sudo nvidia-smi -pm 1",
+      # Format and mount multiple Local SSD
+      "sudo apt update && sudo apt install mdadm --no-install-recommends",
+      "find /dev/ | grep google-local-nvme-ssd",
+      "sudo mdadm --create /dev/md0 --level=0 --raid-devices=16 /dev/disk/by-id/google-local-nvme-ssd-0 /dev/disk/by-id/google-local-nvme-ssd-1 /dev/disk/by-id/google-local-nvme-ssd-2 /dev/disk/by-id/google-local-nvme-ssd-3 /dev/disk/by-id/google-local-nvme-ssd-4 /dev/disk/by-id/google-local-nvme-ssd-5 /dev/disk/by-id/google-local-nvme-ssd-6 /dev/disk/by-id/google-local-nvme-ssd-7 /dev/disk/by-id/google-local-nvme-ssd-8 /dev/disk/by-id/google-local-nvme-ssd-9 /dev/disk/by-id/google-local-nvme-ssd-10 /dev/disk/by-id/google-local-nvme-ssd-11 /dev/disk/by-id/google-local-nvme-ssd-12 /dev/disk/by-id/google-local-nvme-ssd-13 /dev/disk/by-id/google-local-nvme-ssd-14 /dev/disk/by-id/google-local-nvme-ssd-15",
+      "sudo mdadm --detail --prefer=by-id /dev/md0",
+      "sudo mkfs.ext4 -F /dev/md0",
+      "sudo mkdir -p /scratch",
+      "sudo mount /dev/md0 /scratch",
+      "sudo chmod a+w /scratch",
+      "cd /scratch",
+      # Prepare data
+      "mkdir -p models",
+      "mkdir -p preprocessed_data",
+      "gsutil -m cp -n -r gs://tohaowu/mlpinf-v40/models/Llama2 models/",
+      "gsutil -m cp -n -r gs://tohaowu/mlpinf-v40/preprocessed_data/open_orca preprocessed_data/",
+      "mv models/Llama2/fp8-quantized-ammo/llama2-70b-chat-hf-tp2pp1-fp8/ models/Llama2/fp8-quantized-ammo/llama2-70b-tp2pp1-fp8/",
+      "git clone https://github.com/mlcommons/inference_results_v4.0",
+      "cd /scratch/inference_results_v4.0/closed/Google",
+      "export MLPERF_SCRATCH_PATH=/scratch",
+      "cp /scratch/inference_results_v4.0/closed/{NVIDIA,Google}/Makefile.docker",
+      "sed -i '27i\ARCH=x86_64' Makefile",
+      "sed -i '29i\ARCH=x86_64' Makefile.docker",
+      "sudo usermod -a -G docker $USER",
+      # Build and launch a docker container
+      "make prebuild DOCKER_DETACH=1",
```

Comment:
How long does build take? I remember it took a while in the past.

Reply:
Yes it took a while - around 1h

Reply:
Similar to the other comment, is there a way to use saved images/snapshots/etc. to shorten this time? If not, then so be it, but just want to strike a balance between using the most up-to-date software, vs, executing reasonably quick runs.